test.js

```diff
@@ -50,11 +50,10 @@ function test (name, fn, opts) {
 }
 
 function db2arr (t, db, callback, opts) {
-  db.createReadStream(opts)
-    .pipe(listStream.obj(function (err, arr) {
-      if (err) return t.fail(err)
-      callback(arr)
-    }))
+  concat(db.iterator(opts), function (err, arr) {
```

Comment:
I assume we're indirectly depending on the latest version of `levelup`, which exposes `.iterator`?

Reply:
Yep!

![screenshot from 2018-07-05 00-56-48](https://user-images.githubusercontent.com/308049/42295797-5d8a5a74-7fee-11e8-95e6-0eee06065893.png)